src/pages/my_page/user_profile/ResearchProfilePanel.tsx

```diff
@@ -65,13 +70,16 @@ export const ResearchProfilePanel = ({ person, isLoadingPerson }: ResearchProfil
               {t('my_page.my_profile.research_profile_summary.affiliations')}
             </Typography>
             <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem' }}>
-              {activeAffiliations.map((affiliation) => (
-                <AffiliationHierarchy key={affiliation.organization} unitUri={affiliation.organization} />
+              {activeAffiliations.map((affiliation, index) => (
+                <div key={index}>
+                  <AffiliationHierarchy key={affiliation.organization + index} unitUri={affiliation.organization} />
+                  <Divider key={index} />
+                </div>
```

Comment:
Ønsker vi en ekstra div her, eller er det like greit uten?
```suggestion
                <Fragment key={index}>
                  <AffiliationHierarchy key={affiliation.organization + index} unitUri={affiliation.organization} />
                  <Divider key={index} />
                </Fragment>
```

Reply:
Det var bare for å få på plass `key` :+1: